<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>625</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>975</width>
    <height>625</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>975</width>
    <height>625</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="msgTextEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>781</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="msgBrowser">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>781</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="Frame">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>781</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/20220808141751.jpg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFontComboBox" name="fontCbx">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sizeCbx">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>7</number>
       </property>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>17</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>21</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>22</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>23</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="italicTBtn">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolTip">
        <string>斜体</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/images/xieti.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="boldTBtn">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolTip">
        <string>加粗</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/images/cuti.jpg);</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="underlineTBtn">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolTip">
        <string>下划线</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/images/xiahuaxian.jpg);</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="colorTBtn">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolTip">
        <string>字体颜色</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/images/yanse.jpg);</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/images/jietu.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="sendToolBtn">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolTip">
        <string>发送文件</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/images/send.jpg);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveTBtn">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolTip">
        <string>保存</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/images/save.jpg);</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clearTBtn">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolTip">
        <string>删除</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/images/delete.jpg);</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>590</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 127);</string>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="usrTblWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/XHS_166000877449601027901kn4vl9wgmd1010jnc0h1bnbb.jpg);</string>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <row>
     <property name="text">
      <string>在线</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>用户名</string>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="background">
      <color alpha="0">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>主机名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>IP</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>群成员</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>DESKTOP-LETSQQN</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>192.168.88.1</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="UsrNumLbl">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>580</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>在线人数：0人</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>600</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
